devstack_romana.md

```diff
@@ -0,0 +1,93 @@
+Tenant:c858ac6ab9534f8f86f2fe9a731277e1 (admin)
+Segments: default, s1, s2
+```
+
+We've also created `admin` and `demo` tenants, with segments. If the `admin` user creates a new instance, it needs to provide a named segment.
+This will create an IP address within that segment and configure `iptables` rules to restrict communication to hosts within that segment.
+
+## Create a new instance
+
+When creating an instance, you should provide additional metadata to specify the segment that the instance should be allocated into.
+```sh-session
+ubuntu@ip-192-168-99-10:~$ nova boot --flavor m1.nano --image cirros-0.3.4-x86_64-uec --key-name shared-key --nic net-id=$(neutron net-show romana -Fid -f value) --meta romanaSegment=default example-instance
++--------------------------------------+----------------------------------------------------------------+
+| Property                             | Value                                                          |
++--------------------------------------+----------------------------------------------------------------+
+| ...                                  |                                                                |
+| created                              | 2016-03-02T22:46:38Z                                           |
+| flavor                               | m1.nano (42)                                                   |
+| ...                                  |                                                                |
+| id                                   | 048054fe-ec19-45ff-a679-83c19ebb5026                           |
+| image                                | cirros-0.3.4-x86_64-uec (c11f6cde-7bb8-40ed-bce8-fedf676ef1ff) |
+| ...                                  |                                                                |
+| metadata                             | {"romanaSegment": "default"}                                   |
+| name                                 | example-instance                                               |
+| ...                                  |                                                                |
++--------------------------------------+----------------------------------------------------------------+
+ubuntu@ip-192-168-99-10:~$ nova list
++--------------------------------------+------------------+--------+------------+-------------+------------------+
+| ID                                   | Name             | Status | Task State | Power State | Networks         |
++--------------------------------------+------------------+--------+------------+-------------+------------------+
+| 048054fe-ec19-45ff-a679-83c19ebb5026 | example-instance | ACTIVE | -          | Running     | romana=10.0.17.3 |
++--------------------------------------+------------------+--------+------------+-------------+------------------+
+
+```
+
+A new instance was created, and has IP `10.0.17.3`. You can SSH into this from the host it was created on.
```

Comment:
"... and the IP address 10.0.17.3 was assigned to it. You can SSH into this from the host it was created on. We can even see by the IP address's network prefix (10.0/16) that this instance was created on host ip-192-168-99-10 (remember the romana show-host... output shown earlier)."